sdk/storage/AzureStorageBlob/Source/DataStructures/TransferCollection.swift

```diff
@@ -96,12 +96,14 @@ public struct TransferCollection {
     ///   uploads this is the source.
     ///   - state: The current state of the transfer.
     public func firstWith(
-        containerName _: String? = nil,
-        blobName _: String? = nil,
-        localUrl _: LocalURL? = nil,
-        state _: TransferState? = nil
+        containerName: String? = nil,
+        blobName: String? = nil,
+        localUrl: LocalURL? = nil,
+        state: TransferState? = nil
     ) -> BlobTransfer? {
-        return items.first { match($0) }
+        return items.first { transfer in
```

Comment:
Good catch! That's sort of the one thing I didn't like about making all of the matching arguments optional, is that the autocomplete suggests `transfers.firstWith()` as an option which obviously makes no sense, but there's no way to encode "all of these arguments are optional but at least one must be specified".

That said, I made the same mistake above in `filterWhere` - mind fixing it up there as well? Thanks <3

Reply:
Done 👍